nemo/collections/asr/models/ssl_models.py

```diff
@@ -469,7 +469,10 @@ def training_step(self, batch, batch_nb):
             spectrograms, spec_masks, encoded, encoded_len, targets, target_lengths
         )
 
-        tensorboard_logs = {'learning_rate': self._optimizer.param_groups[0]['lr']}
+        tensorboard_logs = {
```

Comment:
Do we know why loss value is missing here?

Reply:
Probably attached below